lithops/localhost/localhost.py

```diff
@@ -41,15 +41,16 @@ class LocalhostHandler:
     """
 
     def __init__(self, localhost_config):
-        logger.debug('Creating Localhost compute client')
+        logger.info('Creating Localhost compute client')
         self.config = localhost_config
         self.runtime = self.config['runtime']
+        self.local_runtime_load = self.config.get('local_runtime_load', False)
```

Comment:
@gilv I think that maybe this variable should be True by default

Reply:
@JosepSampe it's a bit complicated process to create image + runtime. Not sure if majority will do it, even it greatly improve performance. When it false, Lithops will deploy runtime as always